htdocs/core/lib/company.lib.php

```diff
@@ -1248,27 +1257,27 @@ function show_actions_done($conf, $langs, $db, $filterobj, $objcon = '', $noprin
         $sql.= " LEFT JOIN ".MAIN_DB_PREFIX."user as u on u.rowid = a.fk_user_action";
         $sql.= " LEFT JOIN ".MAIN_DB_PREFIX."c_actioncomm as c ON a.fk_action = c.id";
 
+        $force_filter_contact = FALSE;
```

Comment:
php checkstyle will report error if we use False with FALSE.
We must use case False and True.